src/objects/UnsupportedStatistic.ts

```diff
@@ -0,0 +1,23 @@
+import { createUnsupportedStatWidget } from "../window/createUnsupportedStatWidget";
+import { Statistic } from "./Statistic";
+
+export class UnsupportedStatistic extends Statistic<any, any> {
+    constructor(statistic: Statistic<any, any>) {
+        super(
+            statistic.statKey,
+            statistic.statName,
+            statistic.resetValue,
+            statistic.minimumApiVersion,
+            () => {},
+            () => {},
+            () => "",
+        );
```

Comment:
I don't think `super` needs to be called here. It would initialize the statistic, creating subscriptions that wouldn't do anything (and potentially undefined behavior depending on why a statistic's `minimumApiVersion` is set). Ideally, a statistic shouldn't need to be initialized at all, and the parameters can be the minimum version and the name.

Reply:
There's a TS rule that you have to call super in the constructor before doing any other property assignment. As a result, I've moved the subscription initialization out of the constructor so that calling super assigns the props but doesn't set up any unnecessary subscriptions.